stable/grafana/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: grafana
-version: 1.20.1
+version: 1.20.2
```

Comment:
please bump to 1.21.0 this is a new feature

Reply:
Done.